### ISSUE TITLE
Screen Reader: Many Pages: The search field is grouped with the button 'x' to clear the text

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77469]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS, open the app
2. Swipe right to focus the Search button and double-tap to activate it
3. Swipe right to focus the Search field and enter any text
4. Try swiping to focus the Clear (x) button in the text field.

This is a global issue occurring in all screens with the Clear button functionality in search fields, e.g.:
1. New Chat
2. Group Chat
3. Reports
4. Workspaces

## Expected Result:
The search field and clear (x) button should be focused and announced separately, e.g.:
- Search field
- Clear button

## Actual Result:
After typing, the field is announced as "Search for anything, Clear".
The "x" button is not individually focusable or operable.

## Area issue was found in:
Search

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Make sure the search field and button 'x' are not grouped, so that each of them is focused and announced separately.
https://appt.org/en/docs/react-native/samples/accessibility-focusability
https://reactnative.dev/docs/accessibility#accessible

## Environment:
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)

## Screenshots/Videos

<img width="1179" height="2431" alt="Image" src="https://github.com/user-attachments/assets/2e4f3379-68f9-4774-8e12-7f89e77ba8cd" />

https://github.com/user-attachments/assets/5c9c346e-8146-42ef-88cd-42df1b657481

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7009057
**Issue reported by:** NA
**Slack conversation:** NA
